### ISSUE TITLE
Add a Yarn command to transpile JavaScript for IE11 testing (web-components branch only)

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/vets-design-system-documentation#563]
Original Issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/401

Referencing the above original issue, we were able to implement a solution that worked to solve the errors that were occurring in `master` branch. At the time of that issue being resolved, Storybook was fully functional in IE11 for the master branch. We were unable, however, to solve the issue of getting it to load the same way in the `web-components` branch. As that ticket was half resolved, we're creating a new issue here to clearly lay out the half of the task that remains to be done, to avoid it being overlooked due to half of the other one being solved. 

At present, the `web-components` branch still gives errors in IE11. The error information we're receiving seems to be along the lines of `NotSupportedError` without a lot of other information. Even with babel settings enabled for the build it seems to still struggle. The desired end result is that Storybook would be usable in the same manner in IE11 on the `web-components` branch as it currently works on `master`.

**Comment by bkjohnson** [comment]
We're combining what was on the `web-components` branch with `master`, so this can probably be closed or modified.

**Comment by bkjohnson** [comment]
Closing since we no longer support IE11.
